getting-started/adding-telerik-ui-to-an-application/installation-adding-application-namespace-declaration.md

```diff
@@ -72,12 +105,15 @@ Or you can select it from the IntelliSense in XAML.![Common Installing Namespace
 	        <telerik:RadTreeView/>
 	    </Grid>
 	</UserControl>
-{{endregion}}
+	{{endregion}}
+	{% endif %}
 
->If you wonder which control in which assembly belongs to, read the main topic about [Controls Dependencies]({%slug installation-installing-controls-dependencies%}).
-
-## See Also
+>tip If you wonder which control in which assembly belongs to, read the main topic about {% if site.site_name == 'Silverlight' %}[Controls Dependencies]({%slug installation-installing-controls-dependencies%}){% else %}[Controls Dependencies]({%slug installation-installing-controls-dependencies-wpf%}){% endif %}.
```

Comment:
We should improve the wording here:
"If you're wondering which controls belongs to which assembly..."